README.md

```diff
@@ -23,16 +23,20 @@ this environment here: https://hub.docker.com/r/2performantirina/middleman-and-i
 
 ## Deploying
 
-Change the directory to `build`, and commit the changes.
+Run `bundle exec middleman build`, which will compile the site to `docs`.
+
+To deploy to staging you can create a PR against https://github.com/RSpec-Staging/rspec-staging.github.io/
+or you can push temporarily with:
 
-To publish to the staging site (http://rspec-staging.github.io/), run:
 ```
-git push staging master
+git push staging <branch>:source --force
```

Comment:
Is `<branch>:` part necessary, isn't it the default?

Reply:
Honestly I'm not sure, I prefer being explicit when forcing.

Reply:
Makes sense. With the variety of `push.default` setting options the behaviour is widely undetermined.

However, manually typing the current branch name is mundane and error-prone, too.
WDYT of:
```
git push --force staging HEAD:source
```

Reply:
WDYT of?
```suggestion
# Deploy your branch to source on staging
git push staging <branch>:source --force
# or if you have your branch checked out
git push staging HEAD:source --force
```

Reply:
> or if you have your branch checked out

I believe it's like 99% of use cases :D